docs/examples/model_ex/model_examples_stacked.py

```diff
@@ -67,6 +67,8 @@
 
 ax_main.legend()
 
-add_luminosity(collaboration="Beast III", ax=ax_main, lumi=200, preliminary=True)
+add_luminosity(
+    collaboration="Plothist", ax=ax_main, lumi=3, lumi_unit="zb", preliminary=True
```

Comment:
I liked showing a space in the example for the collaboration, as people might do unnecessary `\,`  or `$$` modes.